### ISSUE TITLE
DCD crashes quite often with ST3 (Win8.1 x64)

### ISSUE THREAD

**Issue by ParticlePeter** [yazd/DKit#39]
Just would like to know if the following is a known issue with possibly a known fix. 
I had this problem with DMD version from 2.069.0 up to 2.071.0 on Win 8.1 x64. I never tried using it with any other text editor or IDE, hence asking here first.
DCD crashes with this error message:
---------------------------
std.socket.SocketOSException
---------------------------
std.socket.SocketOSException@std\socket.d(2777): Unable to connect socket: No connection could be made because the target machine actively refused it.
----------------
0x004268B3
0x0042136F
0x00421333
0x00421234
0x0041647C
0x77067C04 in BaseThreadInitThunk
0x778AAB8F in RtlInitializeExceptionChain
0x778AAB5A in RtlInitializeExceptionChain

**Comment by yazd** [comment]
I take it that DCD works at times and breaks at others?
I haven't had this exact problem before. But I think this could be a [DCD](https://github.com/Hackerpilot/DCD/) problem where the server crashes on your code. I can't be certain without looking at what is happening.
Anyway, there is a command to restart DCD server from within sublime. You can try that as a workaround.

**Comment by ParticlePeter** [comment]
Yes, it works quite unreliable and I could not figure out when it works and when not. It crashes even after a DKit: Restart DCD server, mostly immediately. Then it somehow suddenly works, no idea why and when. I added the DCD port as custom rule to my firewall and am of the impression that it happens less often, but it still dos happen. Anyway, thanks for your help.
